packages/terra-form-select/src/SelectField.jsx

```diff
@@ -38,6 +39,10 @@ const propTypes = {
    * Whether to hide the required indicator on the label.
    */
   hideRequired: PropTypes.bool,
+  /**
+   * The intl object containing translations. This is retrieved from the context automatically by injectIntl.
```

Comment:
We should add the `@private` syntax so this doesn't show up in the generated prop-types documentation.